components/model.py

```diff
@@ -18,8 +18,9 @@
 INDEX_SECOND_ELEM = 1
```

Comment:
please fix this linting issue: Line 7: standard import "import hashlib" comes before "import web" (wrong-import-order)